### PR TITLE
Alusine - JavaScript3 - Week2

### PR DESCRIPTION
**Pull request by alquinex**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -30,15 +32,97 @@
...
+
+  function fetchAndRenderData(selectedRepo, repoContainer, contributorsContainer, root) {
+    repoContainer.innerHTML = '';
+    contributorsContainer.innerHTML = '';
+
+    const updatedAt = new Date(selectedRepo.updated_at);
+    createAndAppend('p', repoContainer, {
+      text: `Repository: ${selectedRepo.name}`,
+      class: 'repo-child',
+    });
```
Comment (remarcmij) [inline homework/index.js]: This comment from week 1 is still left unimplemented:

> The assignment asks that the repo name is a link that opens the current GitHub repo in a new browser tab.

File: homework/style.css [hunk 2]
```diff
@@ -1,3 +1,104 @@
 .alert-error {
```
Comment (remarcmij) [inline homework/style.css]: Your CSS styling still has room for improvement.

File: homework/index.js (hunk 1 above)
Comment (alquinex) [reply homework/index.js]: Hi @remarcmij, thanks for the feedbacks and sorry I have not responded to the week1 feedback or even implemented the changes. I will try to do so this latest today. And will certainly improve on the CSS in the week3.

Comment (alquinex) [reply homework/index.js]: Hi @remarcmij, all the changes have been made. The rest of the CSS will be handled in week3.

File: homework/style.css (hunk 2 above)
Comment (alquinex) [reply homework/style.css]: Changes made!

**Pull request closed without merging**